File: 백준/Bronze/28290. 안밖？ 밖안？ 계단？ 역계단？/안밖？ 밖안？ 계단？ 역계단？.cs
string S = Console.ReadLine();

if (S == "fdsajkl;" || S == "jkl;fdsa")
{
    Console.WriteLine("in-out");
}
else if (S == "asdf;lkj" || S == ";lkjasdf")
{
    Console.WriteLine("out-in");
}
else if (S == "asdfjkl;")
{
    Console.WriteLine("stairs");
}
else if (S == ";lkjfdsa")
{
    Console.WriteLine("reverse");
}
else
{
    Console.WriteLine("molu");
}

File: 백준/Bronze/28290. 안밖？ 밖안？ 계단？ 역계단？/안밖？ 밖안？ 계단？ 역계단？.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String S = scanner.nextLine();

        if (S.equals("fdsajkl;") || S.equals("jkl;fdsa")) {
            System.out.println("in-out");
        } else if (S.equals("asdf;lkj") || S.equals(";lkjasdf")) {
            System.out.println("out-in");
        } else if (S.equals("asdfjkl;")) {
            System.out.println("stairs");
        } else if (S.equals(";lkjfdsa")) {
            System.out.println("reverse");
        } else {
            System.out.println("molu");
        }
    }
}
